src/modules/nfts/components/wizard/MintInProgress.tsx

```diff
@@ -0,0 +1,334 @@
+    }
+
+    let metaData: UploadedFilePin | null = null;
+
+    try {
+      metaData = await uploadMetaData(nftValue);
+
+      attemptMint(metaData);
+    } catch (err) {
+      setTransactionStep(TransactionStep.META_DATA_UPLOAD_FAILED);
+    }
+  }, [attemptMint, nftValue, uploadMetaData, setTransactionStep]);
+
+  useEffect(() => {
+    if (showErrors) {
+      return;
+    }
+
+    if (isActive && nftValue) {
+      if (transactionStep === TransactionStep.META_DATA_UPLOADING) {
+        attemptMetaDataUpload();
+      } else if (transactionStep === TransactionStep.SIGNING) {
+        attemptHolaplexSign();
+      }
+    }
+  }, [
+    isActive,
+    showErrors,
+    nftValue,
+    transactionStep,
+    attemptHolaplexSign,
+    attemptMetaDataUpload,
+    index,
+  ]);
+
+  if (!nftValue) {
+    return null;
+  }
+
+  return (
+    <NavContainer
+      title={`Minting ${index + 1} of ${images.length}`}
+      previousStep={previousStep}
+      goToStep={goToStep}
+    >
+      <Row>
+        <Col style={{ marginRight: 224 }}>
+          <Row>
+            <Paragraph style={{ fontWeight: 900 }}>Progress</Paragraph>
+          </Row>
+          <Row style={{ marginTop: 37 }}>
+            <Col style={{ minWidth: 237 }}>
+              <MintStep
+                text="Uploading metadata"
+                isActive={transactionStep === TransactionStep.META_DATA_UPLOADING}
+                isDone={transactionStep > TransactionStep.META_DATA_UPLOADING}
+                failed={transactionStep === TransactionStep.META_DATA_UPLOAD_FAILED}
+              />
+              <MintStep
+                text="Approving transaction"
+                isActive={transactionStep === TransactionStep.APPROVING}
+                isDone={transactionStep > TransactionStep.APPROVING}
+                failed={transactionStep === TransactionStep.APPROVAL_FAILED}
+              />
+              <MintStep
+                text="Sending transaction to Solana"
+                isActive={transactionStep === TransactionStep.SENDING}
+                isDone={transactionStep > TransactionStep.SENDING}
+                failed={transactionStep === TransactionStep.SENDING_FAILED}
+              />
+              <MintStep
+                text="Waiting for final confirmation"
+                isActive={transactionStep === TransactionStep.FINALIZING}
+                isDone={transactionStep > TransactionStep.FINALIZING}
+              />
+              <MintStep
+                text="Signing Holaplex as co-creator"
+                isActive={transactionStep === TransactionStep.SIGNING}
+                isDone={transactionStep === TransactionStep.SUCCESS}
+                failed={transactionStep === TransactionStep.SENDING_FAILED}
+              />
+            </Col>
+          </Row>
+          {showErrors && (
+            <>
+              <Row style={{ marginTop: 144 }}>
+                <Paragraph style={{ fontSize: 14, color: '#D24040' }}>
```

Comment:
Note to refactor inline styles to styled component or best is ant theme options.

The Text component has different types that you can piggyback off. One is danger.

Reply:
So happy to use StyledComponents or if there is a corresponding Antd option.  My only hangup is for things that have just one CSS property to change, it's a bit of mental overhead to extract that all the way into a StyledComponent.  At Goodrx they made heavy use of [styled-system](https://styled-system.com/) which allowed you to do things like this

```tsx

<Box
  fontSize={4}
  fontWeight='bold'
  p={3}
  mb={[ 4, 5 ]}
  color='white'
  bg='primary'>
  Hello
</Box>
```


Similar to how antd sets a `justify` prop on `<Row justify="center" />`

In my opinion it just seems easier to throw one CSS attributes on the corresponding JSX element.  I'll go ahead and refactor the more egregious inline styles to styled components.